source/includes/_cvt_events.md

```diff
@@ -7,29 +7,90 @@
 |---|---|
 | GET | /client_reports/cvt_events/[gmaid]?[query_params]|
 
-### API Name: CVT Events
+### API Name: CVT/Web Events
+
+### Summary
+
+This is an API that can be used by customers for whom we are pages of importance to you (CVT/Web Events). All metrics are reported against the CVT/Web Event.
+
+Web Events fall under two categories: 'Request' and 'Submit'. A 'Request' Web Event is usually informational, in some cases, it can have a post-click activity that can generate a lead. A 'Submit' CVT/Web Event is lead-generating. Metrics tied to a 'Request' CVT/Web Event are considered non-qualified, whereas metrics tied to a 'Submit' CVT/Web Event are considered qualified.
```

Comment:
I would quote or italicize "qaulified" and "non-qualified".

Reply:
I've added quotes to these to keep consistency with 'Request' and 'Submit'.